modules/charon-core/src/test/java/org/wso2/charon3/core/protocol/endpoints/GroupResourceManagerTest.java

```diff
@@ -549,15 +590,11 @@ public void testUpdateGroupWithPUTProvidedUserManagerHandlerIsNull(String id, St
         abstractResourceManager.when(()
                 -> AbstractResourceManager.encodeSCIMException(any(InternalErrorException.class)))
                 .thenReturn(getEncodeSCIMExceptionObject(new InternalErrorException()));
-
-        abstractResourceManager.when(() -> userManager.getGroup(id,
+        Mockito.when(userManager.getGroup(id,
                 ResourceManagerUtil.getAllAttributeURIs(schema))).thenReturn(groupOld);
-
         Group validatedGroup = (Group) ServerSideValidator.validateUpdatedSCIMObject(groupOld, groupNew, schema);
-        abstractResourceManager
-        .when(() -> userManager.updateGroup(any(Group.class),
-                any(Group.class), any(Map.class))).thenReturn(validatedGroup);
-
+        Mockito.when(userManager.updateGroup(any(Group.class),
+                any(Group.class), anyMap())).thenReturn(validatedGroup);
```

Comment:
merge this into 1 line

Reply:
Fixed in commit 34423db0111992d6450b72b5cb1f2b50ab84aec0